go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/TykTechnologies/drl
 
 go 1.12
```

Comment:
shall we bump go version?

Reply:
I'd leave it as-is until go builds with <=1.12 are EOL